src/Kunstmaan/GeneratorBundle/Generator/KunstmaanGenerator.php

```diff
@@ -236,7 +241,12 @@ protected function installDefaultPageTemplates($bundle)
         }
 
         // Twig templates
-        $dirPath = sprintf('%s/Resources/views/Pages/Common/', $bundle->getPath());
+        if (Kernel::VERSION_ID >= 40000) {
+            $dirPath = sprintf('%s/../templates/Pages/Common/', $bundle->getPath());
```

Comment:
The whole if/else can be simplified to this one line:
```suggestion
            $dirPath = $this->getTemplateDir($this->bundle) . '/Pages/Common/'';
```